src/main/resources/lib/vendor/dateFns.ts

```diff
@@ -0,0 +1,37 @@
+import { type Locale } from 'date-fns'
+import { default as nb } from 'date-fns/locale/nb'
```

Comment:
Kan man her ta export med en gang?

Reply:
Har prøvd å gjøre om på det. Fikk det ikke til for type, så lot den være.